pyscriptjs/tests/integration/test_01_basic.py

```diff
@@ -43,6 +43,27 @@ def test_python_exception(self):
         assert tb_lines[0] == "Traceback (most recent call last):"
         assert tb_lines[-1] == "Exception: this is an error"
 
+    # moved before `test_python_exception_in_event_handler`
+    # in order, fails if runs after
```

Comment:
this is very weird because the tests should be run in isolation. If it's true, we should investigate why, but I have the feeling that it's just a coincidence. See my comment below

Reply:
@antocuni I am resolving this since we resolved the one below as well.